### PR TITLE
Adjusts the ordering of metaboxes on edit screens. Closes #6

### LINKED ISSUE #6: Move "Menu Manager" down below "Publish"
see also: https://basecamp.com/2612667/projects/9123052/todos/199460316

### PR DESCRIPTION
**Pull request by matt-bernhardt**
This is a bit speculative - I frankly don't have great confidence in what these functions are doing, or why. It appears that there are three different functions, two for post editing and another for page editing.

Consider this a work in progress.

### CODE REVIEW

**Comment by matt-bernhardt** [comment]
This change is ready for review. The big chance is a re-ordering of the list in the `metabox_order()` function in functions.php, which should move the Menu Manager box down the edit screen.

It also does some code cleaning on these functions, to chip away at technical debt.

Review: @PBruk or @frrrances 
FYI: @darcyduke @hartmans7

File: functions.php [hunk 1]
```diff
@@ -762,41 +762,53 @@ function stop_404_guessing($url) {
...
+		'normal' => join(
+			',',
+			array( // Arrange here as you desire.
+				'submitdiv',
+				'pageparentdiv',
+				'prfx_meta',
+				'categorydiv',
+				'tagsdiv-post_tag',
+				'postimagediv',
+				'dmm-meta-box',
```
Comment (matt-bernhardt) [inline functions.php]: This is the line that sets that metabox to the bottom of the display.

File: functions.php [hunk 2]
```diff
@@ -762,41 +762,53 @@ function stop_404_guessing($url) {
...
-    return $result;
+/**
+ * First make all metaboxes have 'normal' context.
+ * If you know the ids of the metaboxes, you could add them here and skip the next function altogether.
+ *
+ * @link https://github.com/byt/byt-wp-plugin-admin/blob/master/byt-wp-plugin-admin.php
+ */
+function one_column_for_all() {
+	$result['normal'] = 'submitdev, postexcerpt,formatdiv,trackbacksdiv,tagsdiv,post_tag,categorydiv,postimagediv,postcustom,commentstatusdiv,slugdiv,authordiv';
+	$result['side'] = '';
```
Comment (matt-bernhardt) [inline functions.php]: I believe this forces the sidebar to be empty.

**Comment by frrrances** [comment]
@matt-bernhardt This fixed it on my view, but did I overhear you saying that this setting might not be relevant or work as expected in some cases? If so, is it worth merging or will it possibly result in strange states down the line?

**Comment by matt-bernhardt** [comment]
Yes, there's a few things I saw late yesterday that make me consider whether this is ready for merging. Ideally I'd like to discuss this in person to show you what I saw - if that ends up pushing this back to the next release, so be it.

**Comment by matt-bernhardt** [comment]
This didn't make it into 1.3.0, so will be re-opened against 1.4.0.

**Pull request closed without merging**